### DISCUSSION TITLE
CTS questions.

### DISCUSSION THREAD

**Discussion by rcaridade145** [KhronosGroup/MoltenVK#1948] [General]
Looking at the output of the cts run i get 3972 hits on 

[mvk-error] VK_ERROR_INVALID_SHADER_NV: Shader module does not contain an entry point named 'main0'.

I noticed this was last seen on https://github.com/KhronosGroup/MoltenVK/issues/645 .

Looking at a bug report on Ryujinx (Switch Emulator)

https://github.com/Ryujinx/Ryujinx/pull/5094

I wonder if this would fix some CTS tests on MoltenVK.

**Comment by billhollings** [comment]
Thanks for posting this. I see the same errors in CTS runs.

I've created issue #1950 to track this.

However...it is not obvious to me how this is related to https://github.com/Ryujinx/Ryujinx/pull/5094. Can you post a comment to the [new issue](https://github.com/KhronosGroup/MoltenVK/issues/1950) to clarify how it is related?

**Comment by billhollings** [comment]
I've investigated this further. Further explanation can be found in the [related issue](https://github.com/KhronosGroup/MoltenVK/issues/1950#issuecomment-1593908830).
